server/build.gradle

```diff
@@ -32,7 +32,8 @@ plugins {
     id('com.google.cloud.tools.jib') version '3.4.4'
 }
 
-sourceCompatibility = 1.8
+sourceCompatibility = 11
+group 'org.apache.cassandra'
```

Comment:
the server group differs from the subprojects. This doesn't change existing behavior. We declare the common group in the parent gradle (it used to be 'org.apache.cassandra') and now we explicitly declare it in the server's build.gradle file